docs/targeting.md

```diff
@@ -101,3 +102,36 @@ EXAMPLES
 
   $ dvc targeting get feature-one environment-one
 ```
+
+## `dvc targeting update [FEATURE] [ENVIRONMENT]`
+
+Update Targeting rules
+
+```
+USAGE
+  $ dvc targeting update [FEATURE] [ENVIRONMENT] [--config-path <value>] [--auth-path <value>] [--repo-config-path
+    <value>] [--client-id <value>] [--client-secret <value>] [--project <value>] [--no-api] [--headless] [--name
+    <value>] [--targets <value>]
+
+ARGUMENTS
+  FEATURE      The Feature for the Targeting Rule.
+  ENVIRONMENT  The Environment where the Targeting Rule will be updated.
+
+FLAGS
+  --name=<value>     Human readable name
```

Comment:
this may be incorrect, the feature config shouldn't have a name associated with it, only the individual targeting rules

Reply:
yup you're right. Will remove